src/editor/navigation.ts

```diff
@@ -74,6 +83,18 @@ export class NavigationHandler {
                 step.line - 1,
                 step.column - 1
             ) : undefined
+        }).then((editor: TextEditor) => {
```

Comment:
Should this be tied to 'Show reproduction steps' (possibly in `diagnostics.ts`) instead of the 'Jump to report'?
That way it would be easier to show/hide on command as well.